### DISCUSSION TITLE
How to use it on js project

### DISCUSSION THREAD

**Discussion by AugustinMauroy** [wooorm/mdxjs-rs#58] [Q&A]
I have a blog built with nextjs and MDX but I would like to use mdxjs-rs as transilator but idk how to use it in js project

**Comment by ChristianMurphy** [comment]
https://nextjs.org/docs/app/api-reference/next-config-js/mdxRs

**Reply by AugustinMauroy** [reply]
It's when your page is in markdown, not when you catch all segments and fs.read and then parse.

**Reply by wooorm** [reply]
Yes.

I don’t think you should do what you want to do: I doubt switching between languages will be faster.

Instead: use files.

If you do want to do what you ask for, that’s a general question: how to use rust crates in node. A search engine or AI or so can answer that.
